.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,55 @@
+# Build and push the docs website
+name: Build docs
```

Comment:
Aaaahh I didnt see that, the thing is that for me as a user reading the docs is like the first step to understand how the collection works. There are docs sections that usually are manually updated (how to contribute and things like that) but there are others like each role parameters and description that makes really easier to jump in and test/contribute.

Maybe we could also automate how the documentation is generated, please take a look at https://docs.openstack.org/tripleo-ansible/latest/roles.html and https://docs.kubeinit.org/roles.html so as long as the code is 'up-to-date' the docs should be also aligned. Also, i.e. the group variables documentation could be easily rendered form the code as in https://docs.kubeinit.org/usage.html#deployment

But still it will be needed a GH action that builds the docs and push it to make it available automatically.